tests/v2/Smoke/Beneficiary/ApplicationAvailabilityFunctionalTest.php

```diff
@@ -0,0 +1,120 @@
+            sprintf('/contact/%d/delete', $contactTodeleteId),
+        ]);
+    }
+
+    public function testNotePages(): void
+    {
+        $noteId = NoteFactory::findOrCreate(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $noteTodeleteId = NoteFactory::createOne(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $beneficiaryId = $this->beneficiary->getId();
+
+        $this->assertRoutes([
+            sprintf('/beneficiary/%d/notes', $beneficiaryId),
+            sprintf('/beneficiary/%d/note/create', $beneficiaryId),
+            sprintf('/note/%d/detail', $noteId),
+            sprintf('/note/%d/edit', $noteId),
+            sprintf('/note/%d/delete', $noteTodeleteId),
+        ]);
+    }
+
+    public function testEventPages(): void
+    {
+        $eventId = EventFactory::findOrCreate(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $eventToDeleteId = EventFactory::createOne(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $beneficiaryId = $this->beneficiary->getId();
+
+        $this->assertRoutes([
+            sprintf('/beneficiary/%d/events', $beneficiaryId),
+            sprintf('/beneficiary/%d/event/create', $beneficiaryId),
+            sprintf('/event/%d/detail', $eventId),
+            sprintf('/event/%d/edit', $eventId),
+            sprintf('/event/%d/delete', $eventToDeleteId),
+        ]);
+    }
+
+    public function testDocumentPages(): void
+    {
+        $documentId = DocumentFactory::findOrCreate(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $documentToDeleteId = DocumentFactory::createOne(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $beneficiaryId = $this->beneficiary->getId();
+        $folderId = FolderFactory::findOrCreate(['beneficiaire' => $this->beneficiary])->object()->getId();
+
+        $this->assertRoutes([
+            sprintf('/beneficiary/%d/documents', $beneficiaryId),
+            sprintf('/document/%d/detail', $documentId),
+            sprintf('/document/%d/rename', $documentId),
+            sprintf('/document/%d/download', $documentId),
+            sprintf('/documents/%d/move/folder/%d', $documentId, $folderId),
+            sprintf('/document/%d/delete', $documentToDeleteId),
+            sprintf('/document/%d/tree-view-move', $documentId),
+        ]);
+    }
+
+    public function testFolderPages(): void
+    {
+        $firstFolderId = FolderFactory::findOrCreate(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $secondFolderId = FolderFactory::findOrCreate(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $folderToDeleteId = FolderFactory::createOne(['beneficiaire' => $this->beneficiary])->object()->getId();
+        $beneficiaryId = $this->beneficiary->getId();
+
+        $this->assertRoutes([
+            sprintf('/folder/%d', $firstFolderId),
+            sprintf('/beneficiary/%d/folder/create', $beneficiaryId),
```

Comment:
It is interesting, it seems that we do not have inconsistent behavior, should not this route be in the beneficiary controller ? Or have the same prefix as others ?